core/server/master/src/main/java/alluxio/master/meta/checkconf/ServerConfigurationRecord.java

```diff
@@ -33,6 +35,9 @@
   /** Set that contains the addresses of lost nodes. */
   private final Set<Address> mLostNodes;
 
+  /** Listeners to call when needs to check configuration. */
+  private final BlockingQueue<Runnable> mRegenerateReportListeners = new LinkedBlockingQueue<>();
```

Comment:
`ServerConfigurationRecord` just records configuration - it shouldn't need to know anything about the configuration checker or configuration reports.

Reply:
According to the discussion, changed to
```
  /** Listeners to call when this store has any changes. */
  private final List<Runnable> mChangeListeners = new ArrayList<>();
```